Dockerfile_deps

```diff
@@ -1,110 +0,0 @@
-ARG CUDA_VERSION=9.0
-ARG OS_VERSION=7
-ARG NPROC=1
-FROM nvidia/cuda:${CUDA_VERSION}-devel-centos${OS_VERSION}
-LABEL maintainer="AliceVision Team alicevision-team@googlegroups.com"
-
-# use CUDA_TAG to select the image version to use
-# see https://hub.docker.com/r/nvidia/cuda/
-#
-# CUDA_TAG=8.0-devel
-# docker build --build-arg CUDA_TAG=$CUDA_TAG --tag alicevision:$CUDA_TAG .
-#
-# then execute with nvidia docker (https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0))
-# docker run -it --runtime=nvidia alicevision
-
-
-# OS/Version (FILE): cat /etc/issue.net
-# Cuda version (ENV): $CUDA_VERSION
-
-ENV AV_DEPS_DEV=/opt/AliceVisionDeps_git \
-    AV_DEPS_BUILD=/tmp/AliceVisionDeps_build \
-    AV_INSTALL=/opt/AliceVision_install \
-    AV_BUNDLE=/opt/AliceVision_bundle \
```

Comment:
@deece Why did you remove AV_BUNDLE?

Reply:
We need it on the Meshroom side.

Reply:
It didn't seem to be needed - Meshroom was happy to run from the install, and we need the install to build Meshroom (the bundle is missing some components needed for compilation)